scene/2d/navigation/navigation_obstacle_2d.cpp

```diff
@@ -71,7 +71,7 @@ void NavigationObstacle2D::_bind_methods() {
 
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "radius", PROPERTY_HINT_RANGE, "0.0,500,0.01,suffix:px"), "set_radius", "get_radius");
 	ADD_PROPERTY(PropertyInfo(Variant::PACKED_VECTOR2_ARRAY, "vertices"), "set_vertices", "get_vertices");
-	ADD_GROUP("NavigationMesh", "");
+	ADD_GROUP("NavigationPolygon", "");
```

Comment:
This is named navigation mesh as a generic category, not related to the 2d navmesh that is named NavigationPolygon, so shouldn't be renamed. We try to have 2d and 3d named navmesh / navigation mesh where possible to simplify documentation and because users are always confused when you mix them.